templates/server.xml.erb

```diff
@@ -46,6 +46,8 @@
 
     <Service name="Catalina">
         <Connector port="<%= @tomcat_port %>"
+                   relaxedPathChars="[]|" 
+                   relaxedQueryChars="[]|{}^&#x5c;&#x60;&quot;&lt;&gt;"
```

Comment:
there are some really strange characters at the end. Is this on purpose?

Reply:
ah okay, that's even mentioned in their docs.

Reply:
I thought that at first but it's exactly what's in the upgrade docs

Reply:
a comment that the characters are ```\`"<>``` would be good along with why you would want that